### PR TITLE
[IMP] - Add multi company management to partner category

### PR DESCRIPTION
**Pull request by sbejaoui**

### CODE REVIEW

File: res_partner_category_multi_company/views/res_partner_category.xml [hunk 1]
```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2019 ACSONE SA/NV
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+
+<odoo>
+
+    <record model="ir.ui.view" id="res_partner_category_form_view">
```
Comment (tbaden) [inline res_partner_category_multi_company/views/res_partner_category.xml]: id before model

use original id (view_partner_category_form)

File: res_partner_category_multi_company/views/res_partner_category.xml [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+        <field name="inherit_id" ref="base.view_partner_category_form"/>
+        <field name="arch" type="xml">
+            <field name="name" position="after">
+                <field name="company_id" groups="base.group_multi_company"/>
+            </field>
+        </field>
+    </record>
+
+
+    <record model="ir.ui.view" id="res_partner_category_tree_view">
```
Comment (tbaden) [inline res_partner_category_multi_company/views/res_partner_category.xml]: id before model

use original id (view_partner_category_list)

**Review by tbaden — COMMENTED** [review]
thanks for the work 👍 
the change itself looks good to me. maybe some optional changes (see below).
some tests and demo data for company(id: 2) would be nice.
tested in runbot as well



<details><summary>travis log</summary>
************* Module res_partner_category_multi_company
res_partner_category_multi_company/__init__.py:1: [C7902(missing-readme), ] Missing ./README.rst file. Template here: https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst
</details>
will be handled by oca bot

**Review by RobinetDenisAcsone — APPROVED** [review]
LGTM

**Comment by qgroulard** [comment]
+1 :+1:

**Review by qgroulard — APPROVED** [review]

**Comment by sbejaoui** [comment]
@OCA/core-maintainers , This PR is ready to merge

**Pull request merged**